download/index.html

```diff
@@ -112,15 +112,8 @@
           <ul class="list-unstyled">
             <li>
               <ul class="list-inline">
-              <li><a href="/release/2.10.3">2.10.3</a></li>
-              <li><a href="/release/2.10.2">2.10.2</a></li>
-                <li><a href="/release/2.10.1">2.10.1</a></li>
-                <li><a href="/release/2.10.0">2.10.0</a></li>
-              </ul>
-              <ul class="list-inline">
-                <li><a href="/release/2.10-RC1">2.10-RC1</a></li>
-                <li><a href="/release/2.10-beta">2.10-beta</a></li>
-                <li><a href="/release/2.10-M0">2.10-M0</a></li>
+                <li><a href="/release/2.11-RC1">2.11-RC1</a></li>
+                <li><a href="/release/2.11-beta">2.11-beta</a></li>
```

Comment:
There should be a 2.11.0 link here too. Looks like whoever was responsible for that release didn't update the website at all.

Presumably some other configuration will be missing as well: https://github.com/geoserver/geoserver.github.io#releases